### DISCUSSION TITLE
GitHub Keyboard Shortcuts Incompatible with Firefox

### DISCUSSION THREAD

**Discussion by GovindKrishnan** [community/community#12869] [Other Feature Feedback, Questions, & Ideas]
Most of the GitHub Keyboard Shortcuts are incompatible with Mozilla Firefox, as it triggers the browser's Quick Find/Page Search Feature. 

 Eg: `g` `p` doesn't work though people are prompted to use it as a Pro Tip.

Noticed `?` works even though Quick Find is triggered, possibly because it's a single key shortcut.

**Comment by jglick** [comment]
Ctrl-Shift-P to preview markdown does not work as Firefox instead launches a private window.

**Reply by mkurz** [reply]
Still not working (using Linux)

**Reply by openendings** [reply]
Still not working (140.3.1esr (64-bit), Debian, packaged as `firefox-esr 140.3.1esr-1~deb13u1`)
